src/upload-sourcemaps.ts

```diff
@@ -79,8 +97,6 @@ module.exports = async (options) => {
         '--ext',
         'map',
         '--rewrite',
-        '--strip-prefix',
```

Comment:
think you may have rebased using an older commit? https://github.com/expo/sentry-expo/commit/21ed6d2a75633881881024ce72d66357a46a5da1

Reply:
there are other instances like that, I'd re-pull master again and then rerebase